### PR TITLE
Vdb 1120 mainnet release

### PR DESCRIPTION
**Pull request by yaoandrew**

### CODE REVIEW

File: transformers/integration_tests/vat_rely_test.go [hunk 1]
```diff
@@ -54,11 +54,11 @@ var _ = Describe("Vat Rely transformer", func() {
...
 		Expect(err).NotTo(HaveOccurred())
 
-		usrAddress := "0x0e4725db88bb038bba4c4723e91ba183be11edf3"
+		usrAddress := "0xbaa65281c2fa2baacb2cb550ba051525a480d3f4"
 		usrAddressID, usrAddressErr := shared.GetOrCreateAddress(usrAddress, db)
 		Expect(usrAddressErr).NotTo(HaveOccurred())
 
-		Expect(len(dbResult)).To(Equal(1))
+		Expect(len(dbResult)).To(Equal(2))
 		Expect(dbResult[0].Usr).To(Equal(usrAddressID))
```
Comment (rmulhol) [inline transformers/integration_tests/vat_rely_test.go]: Are we confident that the `usr` is the same for both events in this block?

File: transformers/integration_tests/vow_flog_test.go [hunk 2]
```diff
@@ -30,7 +30,7 @@ import (
 )
 
 // TODO: replace block number when there is a flog event on the updated Vow
```
Comment (rmulhol) [inline transformers/integration_tests/vow_flog_test.go]: Can probably delete this comment ✂️

File: db/schema.sql [hunk 3]
```diff
@@ -18516,273 +18516,273 @@ CREATE INDEX transactions_header ON public.transactions USING btree (header_id);
 -- Name: flap_bid_bid flap_bid_bid; Type: TRIGGER; Schema: maker; Owner: -
 --
 
-CREATE TRIGGER flap_bid_bid AFTER INSERT OR UPDATE ON maker.flap_bid_bid FOR EACH ROW EXECUTE FUNCTION maker.insert_updated_flap_bid();
+CREATE TRIGGER flap_bid_bid AFTER INSERT OR UPDATE ON maker.flap_bid_bid FOR EACH ROW EXECUTE PROCEDURE maker.insert_updated_flap_bid();
```
[the remark this answers is not in the document]
Comment (yaoandrew) [reply db/schema.sql]: Yeah... looks like this went away after I upgraded my postgres version.

File: db/schema.sql [hunk 4]
```diff
@@ -18516,273 +18516,273 @@ CREATE INDEX transactions_header ON public.transactions USING btree (header_id);
...
-CREATE TRIGGER urn_art AFTER INSERT OR DELETE OR UPDATE ON maker.vat_urn_art FOR EACH ROW EXECUTE FUNCTION maker.update_urn_arts();
+CREATE TRIGGER urn_art AFTER INSERT OR DELETE OR UPDATE ON maker.vat_urn_art FOR EACH ROW EXECUTE PROCEDURE maker.update_urn_arts();
 
 
 --
 -- Name: vat_urn_ink urn_ink; Type: TRIGGER; Schema: maker; Owner: -
 --
 
-CREATE TRIGGER urn_ink AFTER INSERT OR DELETE OR UPDATE ON maker.vat_urn_ink FOR EACH ROW EXECUTE FUNCTION maker.update_urn_inks();
+CREATE TRIGGER urn_ink AFTER INSERT OR DELETE OR UPDATE ON maker.vat_urn_ink FOR EACH ROW EXECUTE PROCEDURE maker.update_urn_inks();
```
[the remark this answers is not in the document]
Comment (yaoandrew) [reply db/schema.sql]: These are gone now with the postgres update

File: transformers/integration_tests/vat_rely_test.go (hunk 1 above)
Comment (yaoandrew) [reply transformers/integration_tests/vat_rely_test.go]: Its different in each event in this block but I've updated the test incase the ordering is off.

File: transformers/integration_tests/plugin_test.go [hunk 5]
```diff
@@ -144,7 +144,7 @@ var _ = Describe("Plugin test", func() {
 	viper.SetConfigName("testing")
 	viper.AddConfigPath("$GOPATH/src/github.com/makerdao/vdb-mcd-transformers/environments/")
 	var ilk = "0x4554482d41000000000000000000000000000000000000000000000000000000"
-	var blockNumber = int64(14764569)
+	var blockNumber = int64(8928180) //needs a mainnet block with a cat file flip
```
Comment (elizabethengelman) [inline transformers/integration_tests/plugin_test.go]: 🏅

File: transformers/integration_tests/pot_cage_test.go [hunk 6]
```diff
@@ -15,7 +15,8 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("PotCage EventTransformer", func() {
+//TODO: Need tu update when there are events on mainnet
```
Comment (elizabethengelman) [inline transformers/integration_tests/pot_cage_test.go]: tiny typo: `:%s/tu/to`

**Pull request merged**